# Funciones para modelar la relación entre variables

#### Ejercicio 2: 

El costo por metro lineal de una cerca es de $35$ dólares. Si queremos cercar un terreno rectangular de $25$ metros cuadrados, determina una expresión que calcule el costo total en función del ancho del rectángulo.

#### Solución

* Datos: 

$P$: cantidad de metros de cerca que quiero comprar

$C$: costo total de la cerca -------------------------  variable dependiente

$a$: ancho ----------------------------------- variable independiente

$\ell$: largo

$A$: área, $A=25$

* Fórmula:
    * $A=a\cdot\ell$
    * $P=a+\ell+a+\ell=2a+2\ell$
    * $C=35P$ (lineal)

* Sustitución:
    * $25
    =a\cdot\ell$
    * $P=2a+2\ell$
    * $C=35P$ (lineal)

Queremos despejar la variable que **no** es la independiente.

$$25=a\cdot\ell$$

$$\dfrac{25}{a}=\ell$$

Ahora sustiuimos en otra relación

$$P=2a+2\ell$$

$$P=2a+2\left(\dfrac{25}{a}\right)$$

Ahora sustiuimos en otra relación. Lo podemos hacer porque la ecuación ya es explícita.

$$C=35P$$

$$C(a)=35\left[2a+2\left(\dfrac{25}{a}\right)\right]=70a+\dfrac{1750}{a}$$

#### Ejercicio 8: 

Una hoja de papel debe tener $18\ \rm{cm}^2$ de texto impreso. Además, sus márgenes superior e inferior deben medir $2\ \rm{cm}$ de alto y los márgenes laterales $1\ \rm{cm}$ de anchura. Encuentra la expresión que modela el área de la hoja en función de su base.

#### Solución:

$A_\text{texto}=(b-2)(h-4)$

$A_\text{hoja}=b\cdot h$

$b:$ base de la hoja

$h:$ altura de la hoja

$18=(b-2)(h-4)$

Despejar a la variable que no es la independiente:

$18=(b-2)(h-4)$

$\dfrac{18}{b-2}+4=h$

Sustituimos en la variable que no es la independiente:

$A_\text{hoja}=b\cdot h$

$$A_\text{hoja}=b\cdot\left(\dfrac{18}{b-2}+4\right) $$

Mi función:

$$A_\text{hoja}(b)=b\cdot\left(\dfrac{18}{b-2}+4\right) $$

# Clase 4

## Funciones cuadráticas

$$ f(x)=ax^2+bx+c $$

Al igual que en las funciones lineales, el valor de $c$ lo podemos conocer al saber qué sucede en $x=0$. Si no tenemos este conocimiento, extrapolamos de los datos que tengamos.

Para una función polinomial (lineal, cuadrática, cúbica,...) el dominio máximo de la función $(-\infty,\infty).$

### Gráfica de una función cuadrática

Para las funciones cuadráticas

$$ f(x)=ax^2+bx+c $$

tendremos un punto importante conocido como vértice y que tiene coordenadas

$$ V\left(-\dfrac{b}{2a},c-\dfrac{b^2}{4a}\right) $$

**1er caso:** $a>0$: parábola que abre hacia arriba (tiene un valor mínimo)

Entonces el rango de la función es

$$ \left[c-\dfrac{b^2}{4a},\infty\right) $$

**2do caso:** $a<0$: parábola que abre hacia abajo (tiene un valor máximo)

Entonces el rango de la función es

$$ \left(-\infty,c-\dfrac{b^2}{4a}\right] $$

### Forma de la función cuadrática

Si sabemos que la función cuadrática pasa por el eje de horizontal en dos puntos ($B$ y $C$), podemos decir que la forma de la función cuadrática es

$$ f(x)=A(x-B)(x-C) $$

## Situación. Gravedad en Marte

La exploración de Marte ha requerido del uso de nuevas estrategias a fin de superar
obstáculos con los que se han enfrentado los vehículos destinados a navegar por el
planeta. Algunos de los retos son: el conseguir que los vehículos exploradores
recorran más terreno en menos tiempo, evitando el problema de corrosión de las
ruedas, las tormentas de arena que bloquean los paneles solares y el cambio en las
estaciones del clima en Marte que modifican la incidencia de rayos solares sobre
los paneles.

Para resolver el problema de cubrir más área explorada en menos tiempo, se han
propuesto nuevas ideas como el uso de helicópteros en este planeta o robots
denominados PUH (por sus siglas en inglés) que significa "Saltador Urbano de
Precisión", robots que son capaces de saltar, ahorrando energía en comparación
con volar.

La función que describe el lanzamiento vertical de un objeto como función del
tiempo, es de gran utilidad al momento de determinar la posición del robot PUH
dada su altura inicial, en cualquier cuerpo celeste del que se conoce su gravedad.
Cuando un objeto se lanza hacia arriba desde una altura inicial $h_0$, con una
velocidad inicial $v_0$, la función para determinar su altura es $H(t)$ en el tiempo $t$ está dada por

$$H(t)=h_0+ v_0t −\dfrac{1}{2}gt^2$$

donde $g$ es el valor de la aceleración gravitacional de Marte.

En el esquema de la figura se ilustra el salto en vertical de un robot desde
una altura inicial $h_0$, con una velocidad inicial $v_0$, que alcanza una altura máxima $H(t_\text{max})=\text{max}$, en el tiempo $t_\text{max}$ , para después caer a la superficie del planeta, alcanzando así una altura final $H(t_f)=0$, en el tiempo final $t_f$.


<img src="marte1.png" align="middle" width="400">

**1. El modelo que describe la altura del objeto en lanzamiento vertical como función
del tiempo es una:**

A) Función de primer grado

B) Función de segundo grado

C) Función trigonométrica

D) Función exponencial

**Solución:** B)

**2. ¿Cuál es el dominio máximo de definición de la función?**


A) $(-\infty,\infty)$

B) $[0,15)$

C) $(0,\infty)$

D) $[0, 360)$

**Solución:** A)

**3. Si en el planeta Marte, desde una plataforma situada a una altura $h_0=1.5\ \rm{m}$ un robot salta verticalmente con una velocidad inicial $v_0=20\ \dfrac{\rm{m}}{\rm{s}}$ y al considerar que la aceleración gravitacional en Marte es de $g=3.71\ \dfrac{\rm{m}}{\rm{s}^2}$.**

**¿Cuál es la altura en metros, que el robot alcanza a los dos segundos de su lanzamiento hacia arriba?**

$$H(t)=h_0+ v_0t −\dfrac{1}{2}gt^2$$

$$H(t)=1.5+ 20t −1.855t^2$$

$$H(2)=1.5+ 20(2) −1.855(2)^2$$

In [2]:
1.5+20*(2)-1.855*(2)^2

34.08

A) 45.92

B) 34.08

C) 31.08

D) 48.92

**Solución:** B)

**4. Considerando las circunstancias en la pregunta anterior ¿Cuál es el tiempo
aproximado, en segundos en el que el robot toca la superficie del planeta después
de su salto vertical desde la plataforma?**

$$H(t)=1.5+ 20t −1.855t^2$$

<img src="marte1.png" align="middle" width="400">

Ya sabemos que si llega al piso la altura es cero.

$$0=1.5+ 20t −1.855t^2$$

$$at^2+bt+c=0$$

$a=-1.855$, $b=20$, $c=1.5$

$$ t=\dfrac{-b\pm\sqrt{b^2-4ac}}{2a} $$

$t_1=10.85,\ t_2=-0.07$

A) 5.46

B) 5.31

C) 10.70

D) 10.856

**Solución:** D)

**5. De las siguientes gráficas, ¿cuál representa un esbozo de la gráfica de la función correspondiente al lanzamiento vertical descrito en la pregunta 3?**

$$H(0)=1.5+ 20(0) -1.855(0)^2$$

$$H(0)=1.5$$

Esto me da las coordenadas $(0,1.5)$

C)
<img src="cuadratica5C.png" align="middle" width="400">

D)
<img src="cuadratica5D.png" align="middle" width="400">

**Solución:** D)

**6. La figura muestra la gráfica de la expresión matemática que describe la altura del robot en función del tiempo, cuando éste realiza un salto vertical desde la luna, donde hay una aceleración gravitacional $g=1.6\ \dfrac{\rm{m}}{\rm{s}^2}$ , inicia dicho salto a nivel del suelo y a una velocidad inicial de $v_0=40\ \dfrac{\rm{m}}{\rm{s}}$.**

**¿Cuáles son los puntos A y B de dicha gráfica?**


$$H(t)=h_0+ v_0t −\dfrac{1}{2}gt^2$$

$g=1.6$, $v_0=40$, $h_0=0$

$$H(t)=40t −0.8t^2$$

Hicimos que la altura fuese cero para ver a cuáles tiempos sucede esto.

$$0=40t −0.8t^2$$

$$0=t(40-0.8t)$$

1ra opción: $t=0$

2da opción: $40-0.8t=0$

$$ 40-0.8t=0 $$

$$ -0.8t=-40 $$

$$ t=\dfrac{-40}{-0.8}=50 $$

$(0,0)$ y $(50,0)$

<img src="marte2.png" align="middle" width="500">


B) $A(0,0),\ B(50,0)$

C) $A(0,0),\ B(25,0)$



**Solución:** B)

**7. De acuerdo con la información de la gráfica asociada con la gráfica de la figura anterior, ¿cuál es el dominio natural de la función?**

A) $[0, 25]$

B) $[25, 0]$

C) $[0, 50]$

D) $[50, 0]$

**Solución:** C)

**8. La siguiente figura muestra la gráfica de la expresión matemática que describe la
altura del robot en función del tiempo, cuando éste realiza un salto vertical desde la
luna, e inicia dicho salto a nivel del suelo.**

**¿Cuál es la expresión matemática que modela la gráfica de la figura?**

<img src="marte3.png" align="middle" width="500">

**1ra forma:**

Tenemos las coordenadas:
* $(0,0)$
* $(62.5,0)$

A) $H ( t ) = 0.8( t − 0)(t − 62.5)=0.8(t^2-62.5t)=0.8t^2-0.8\cdot62.5t$

$$H (0) = 0.8( 0 − 0)(0 − 62.5)=0$$

$$H (62.5) = 0.8(62.5 − 0)(62.5− 62.5)=0$$

B) $H (t) = − 0.8(t − 0)(t − 62.5)=-0.8t^2+0.8\cdot62.5t$

$$H(0) = − 0.8(0 − 0)(0 − 62.5)=0$$

$$H(0) = − 0.8(62.5 − 0)(62.5 − 62.5)=0$$

A) $H ( t ) = 0.8( t − 0)(t − 62.5)$

B) $H (t) = − 0.8(t − 0)(t − 62.5)$

C) $H (t) = − 0.8(t + 0)(t + 62.5)$

D) $H ( t ) = 0.8( t + 0)(t + 62.5)$

**Solución:** B)

**9. La siguiente figura corresponde a la gráfica de la altura de un robot lanzado en la luna, desde una altura de $1.2\ \rm{m}$, como función del tiempo ¿Cuál es la expresión que corresponde a dicha gráfica?**


$$H(t)=h_0+ v_0t −\dfrac{1}{2}gt^2$$

$$H(t)=1.2+ v_0t −0.8t^2$$

<img src="marte4.png" align="middle" width="500">

$(0,1.2)$

$(1,2)$

B) $H ( t ) = − 0.8( t +1)^2 + 2$

$$ H(0)=-0.8(0+1)^2+2=1.2 $$

$$ H(1)=-0.8(1+1)^2+2=-2 $$

D) $H ( t ) = − 0.8( t −1)^2 + 2$

$$ H(0)=-0.8(0-1)^2+2=1.2 $$

$$ H(1)=-0.8(1-1)^2+2=2 $$

**Solución:** D)

**10. ¿Cuál es el recorrido (rango) de la función asociado con el lanzamiento vertical de un robot en el planeta Marte a una altura inicial de $2.145\ \rm{m}$, con una velocidad inicial de $3.71\ \dfrac{\rm{m}}{\rm{s}}$, al considerar el dominio máximo de la misma?**

$$H(t)=2.145+ 3.71t −\dfrac{1}{2}(3.71)t^2$$

$$H(t)=2.145+ 3.71t −1.855t^2$$

$$ f(t)=at^2+bt+c $$

$$ V\left(-\dfrac{b}{2a},c-\dfrac{b^2}{4a}\right) $$

$a=-1.855$, $b=3.71$, $c=2.145$

$$ V\left(-\dfrac{3.71}{2(-1.855)},(2.145)-\dfrac{(3.71)^2}{4(-1.855)}\right)=V(1,4) $$

In [4]:
-3.71/(2*(-1.855))

1.0

In [5]:
2.145-(3.71)^2/(4*(-1.855))

4.0

A) $(-\infty, 4]$

D) $(-\infty, 4)$

**Solución:** A)

**11. ¿Cuál es el recorrido (rango) de la función considerando el dominio natural de la misma, para la situación planteada en la pregunta anterior?**

A) $[4,\infty)$

B) $[-\infty,4)$

C) $(0,4)$

D) $[0,4]$

**Solución:** D)

## Logaritmos

* $\log_a(a)=1$

* $\log_a(a)=1$

* $\log_a(x)+\log_a(y)=\log_a(xy)$

* $\log_a(x)-\log_a(y)=\log_a\left(\dfrac{x}{y}\right)$

* $\log_a(x^n)=n\log_a(x)$

* $\log_a(b)=\dfrac{\log_c(b)}{\log_c(a)}$

$$y=x^2\quad \leftrightarrow\quad\sqrt{y}=x $$

$$y=\log_a(x)\quad \leftrightarrow\quad a^y=x $$

* ***Ejemplo:*** Resuelve la ecuación

$$ 3^{x+1}=81 $$

Pasamos el $3$ en la base del otro lado de la ecuación. Pasa como logaritmo:

$$ x+1=\log_3(81) $$

$$ x+1=4 $$

$$ x=4-1 $$

$$ x=3 $$

* ***Ejemplo:*** Encuentra $t$ de tal forma que

$$2^t = 7$$

$$t = \log_2(7)$$

* $\log_a(b)=\dfrac{\log_c(b)}{\log_c(a)}$

$a=2$, $b=7$, $c=10$

$$\log_2(7)=\dfrac{\log_{10}(7)}{\log_{10}(2)}=2.80735922...$$

In [13]:
2^2.80735922

7.00002085377764

* ***Ejemplo:*** Resuelve la ecuación

$$ \log_2(x^7)=21 $$

* $\log_a(x^n)=n\log_a(x)$

$$ 7\log_2(x)=21 $$

$$ \log_2(x)=\dfrac{21}{7}=3 $$

$$ x=2^3=8 $$

* ***Ejemplo:*** Resuelve la ecuación

$$ \log_2(x+1)+\log_2(x-1)=4 $$

* $\log_a(x)+\log_a(y)=\log_a(xy)$

$$ \log_2[(x+1)(x-1)]=4 $$

$$ \log_2(x^2-1)=4 $$

$$ x^2-1=2^4 $$

$$ x^2=2^4+1 $$

$$ x^2=17 $$

$$ x=\sqrt{17} $$

## Funciones exponenciales

$$ f(x)=a\cdot b^x+c $$

* Si $b>1$, decimos que se trata de un crecimiento exponencial.
* Si $b<1$, decimos que se trata de un decaimiento exponencial.
* Si $b=1$, línea horizontal

$a=1$, $b=2$, $c=0$

$$ f(x)=2^x $$

Se dice hace mucho tiempo había inventado el ajedrez. Se lo ofrece a un rey.

El ajedrez tiene 8 filas y 8 columnas (64 casillas).

$$ f(-10)=2^{-10}=\dfrac{1}{2^{10}}=\dfrac{1}{1024} $$

2+4+8+16

In [19]:
2^(-10)

0.0009765625

In [15]:
2^30

1073741824

In [17]:
2^60

1152921504606846976

El dominio máximo de estas funciones es $\mathbb{R}$.

Si $a>0$, el rango de la función es $(c,\infty)$.

Si $a<0$, el rango de la función es $(-\infty,c)$.

### Tasa de crecimiendo

Se define como $b-1$:

* Si $b-1$ es positivo decimos que la función está creciendo $(b-1)\cdot100\%$
* Si $b-1$ es negativo decimos que la función está decayendo $(b-1)\cdot100\%$

$$ f(x)=2^x $$

$b-1=2-1=1$

La función está creciendo $1\cdot100\%$

* ***Ejemplo:*** Pensemos que tenemos una función exponencial $f(x)$. Si $f(20)=88.2$ y $f(23)=91.4$, encuentra la (a) base, y (b) $f(25)$.

## Funciones logarítmicas

$$ f(x)=a\cdot\log(x+b)+c$$

## Situación. Población de Michoacán

En la Tabla se muestra la relación de la población de Michoacán de 1880 a 2010. La tercera
columna nos indica la proporción, la cual fue obtenida de dividir la población de determinada
década entre la población de la década precedente.

| Año  | Población (millones) | Proporción |
|------|----------------------|------------|
| 1880 | 1.48                 | 1.24       |
| 1890 | 1.84                 | 1.24       |
| 1900 | 2.28                 | 1.24       |
| 1910 | 2.82                 | 1.24       |
| 1920 | 3.49                 | 1.24       |
| 1930 | 4.33                 | 1.24       |
| 1940 | 5.36                 | 1.24       |
| 1950 | 6.64                 | 1.24       |
| 1960 | 8.22                 | 1.24       |
| 1970 | 10.18                | 1.24       |
| 1980 | 12.62                | 1.24       |
| 1990 | 15.63                | 1.24       |
| 2000 | 19.35                | 1.24       |
| 2010 | 23.97                | 1.24       |

**1. ¿Cuál de las siguientes afirmaciones es verdadera? [1 punto]**

A) El año es el dominio y la proporción el codominio de la relación.

B) La población es el dominio y la proporción el codominio de la relación.

C) El año es el dominio y la población el codominio de la relación.

D) La población es el dominio y el año el codominio de la relación.

**Solución:** 

**2.  La tasa de crecimiento de 1990 al 2010 de la población de Michoacán es: [2 puntos]**

A) $1.24$

B) $0.417$

C) $2.4$

D) $0.24$

**3. Los datos de la tabla 1 se encuentran modelados por una función del tipo $f(x)=Aa^{Bx}$ donde $A$ y $B$ son constantes, la función es: [1 punto]**

A) Lineal

B) Polinomial

C) Exponencial

D) Logarítmica

## Situación: Fragmentos de Bosque y Hormigas

Como parte de un análisis del efecto de la fragmentación sobre las relaciones
especies-área, se registró el número de especies de hormigas para ciertos
fragmentos de bosque seco tropical. En el siguiente gráfico se presenta la relación
entre el área de los fragmentos de bosque seco tropical en la cuenca media del río
Cauca, Colombia, medida en hectáreas (h) , con el número de especies de hormigas
encontradas en cada fragmento N (h) .


**1. Es el tipo de función que mejor describe la asociación entre el área y el número
de especies que se representa en la figura.**

<img src="bosque1.png" align="middle" width="500">

A) $N(h)=13e^h+76$

B) $N(h)=76e^h+13$

A) $N(h)=13\log(h)+76$

D) $N(h)=76\log(h)+13$

**Solución:** 

**2. Si el modelo que ajusta a los datos planteados en la situación para otra zona de
bosque es $N(h)=7.9\cdot\ln(h)+100$, ¿cuál es el dominio máximo de la función?**

A) $(0,\infty)$

B) $(-\infty,\infty)$

C) $[7.9, 100]$

D) $[100, \infty)$

**3. Si para una zona especial la relación descrita en la situación está dada por $N (h) = 11.37\cdot\ln(h ) + 75.9$, ¿cuál es el despeje para poder determinar el número de hectáreas de un fragmento de bosque donde se encuentran $89$ especies de hormigas?**

A) $h=e^{\dfrac{89-75.9}{11.37}}$

B) $h=\ln\left(\dfrac{89-75.9}{11.37}\right)$

C) $h=\ln\left(\dfrac{11.37}{89-75.9}\right)$

D) $h=e^{\dfrac{11.37}{89-75.9}}$

## ¿Quién gana?

## Situación. El área de un trapecio rectángulo con vértices en dos circunferencias

Si se tienen dos círculos tangentes en el punto $P_T$ con diámetros $\overline{P_TD}=10\ \rm{cm}$ y $\overline{P_TC}=5\ \rm{cm}$, respectivamente, como se muestra en la figura siguiente

<img src="situacion_circulos.png" align="middle" width="300">

**1. Expresar la altura $h$ en términos de la base mayor $x$ del trapecio [3 puntos]**

A) $h(x)=\dfrac{\sqrt{100-x^2}}{4}$

B) $h(x)=\sqrt{\dfrac{100-x^2}{4}}$

C) $h(x)=\dfrac{\sqrt{100+x^2}}{4}$

D) $h(x)=\sqrt{\dfrac{100+x^2}{4}}$

 **2. Expresar la base menor y en términos de la base mayor: [1 punto]**

A) $y=\sqrt{\dfrac{x}{2}}$

B) $y=\dfrac{\sqrt{x}}{2}$

C) $y=\dfrac{x}{2}$

D) $y=\dfrac{x}{4}$

**3. El promedio de las dos bases de trapecio es: [1 punto]**

A) $\dfrac{2x+\sqrt{x}}{4}$

B) $\dfrac{\sqrt{2}x+\sqrt{x}}{2\sqrt{2}}$

C) $\dfrac{5}{4}x$

D) $\dfrac{3}{4}x$

**4. El área del trapecio en términos de la base mayor es: [3 puntos]**

A) $A_T(x)=\dfrac{3x\sqrt{100-x^2}}{16}$

B) $A_T(x)=\dfrac{3x\sqrt{100+x^2}}{16}$

C) $A_T(x)=\dfrac{3x\sqrt{100+x^2}}{4}$

D) $A_T(x)=\dfrac{3x\sqrt{100-x^2}}{4}$